modules/mod_ginger_foundation/lib/css/__default-screen.scss

```diff
@@ -1,98 +1,22 @@
-// If you have globbing
-
-// Global
-
-@import '../../../../../modules/mod_ginger_foundation/lib/css/src/variables';
-@import '../../../../../modules/mod_ginger_foundation/lib/css/src/mixins';
-@import '../../../../../modules/mod_ginger_foundation/lib/css/src/base';
-@import '../../../../../modules/mod_ginger_foundation/lib/css/src/icons';
-@import '../../../../../modules/mod_ginger_foundation/lib/css/src/buttons';
-@import '../../../../../modules/mod_ginger_foundation/lib/css/src/form';
-
-//Vendors
-@import '../../../../../modules/mod_ginger_base/lib/css/vendors/jquery.fancybox';
-@import '../../../../../modules/mod_ginger_base/lib/css/vendors/slick';
-
-// Blocks
-@import '../../../../../modules/mod_ginger_foundation/lib/css/src/blocks/*';
-
-// Layout
-@import '../../../../../modules/mod_ginger_foundation/lib/css/src/layout/*';
-
-// Pages
-@import '../../../../../modules/mod_ginger_foundation/lib/css/src/pages/*';
-
-// Otherwise, these are te
-
 // Global
-@import '../../../../../modules/mod_ginger_foundation/lib/css/src/variables';
-@import '../../../../../modules/mod_ginger_foundation/lib/css/src/mixins';
-@import '../../../../../modules/mod_ginger_foundation/lib/css/src/base';
-@import '../../../../../modules/mod_ginger_foundation/lib/css/src/icons';
-@import '../../../../../modules/mod_ginger_foundation/lib/css/src/buttons';
-@import '../../../../../modules/mod_ginger_foundation/lib/css/src/buttons';
+@import '/lib/css/src/variables';
+@import '/lib/css/src/mixins';
+@import '/lib/css/src/base';
+@import '/lib/css/src/icons';
+@import '/lib/css/src/buttons';
+@import '/lib/css/src/buttons';
```

Comment:
Note from review with Colin: We are curious about how abs/rel paths work here (What is the effect of using the leading slash?); maybe to avoid any confusion we can consistently use relative paths.

Reply:
So this file `__default-screen.scss` was used as an example for sites that want to use foundation but not everything. And we needed to use the absolute paths because of the makefile or something. I guess because we stopped working with 'globbing'?

Reply:
The reason to stop globbing is because it was not supported anymore in newer sass versions.
Aha, thanks for the clarification, we suspected it to be meant as an example indeed 👍